lib/components/form/switchButton.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component, PropTypes } from 'react'
```

Comment:
We seem to be naming files in hear with dashes `fil-name.js`, rather than camelCase.  Any reason it was switched up here?